contracts/transfer/circuits/src/gadgets/range.rs

```diff
@@ -0,0 +1,52 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use dusk_plonk::prelude::*;
+use plonk_gadgets::AllocatedScalar;
+
+/// This gadget simply wraps around the composer's `range_gate` function,
+/// but takes in any type that implements the traits of the note,
+/// for ease-of-use in circuit construction.
+pub fn range(composer: &mut StandardComposer, value: AllocatedScalar) {
+    composer.range_gate(value.var, 64);
```

Comment:
The range has to be always at 64bits?

Reply:
Well, its actually up to u32, as that is the size of the BLsScalar